### PR TITLE
Add shadow design tokens to themes [FEC-903]

### PR DESCRIPTION
**Pull request by noranda**
## What did we change?
- [add shadow tokens](https://github.com/ezcater/recipe/commit/a8c81950e0926d3b05af8d91982d1a23c492c215)
- [add docs for shadow tokens](https://github.com/ezcater/recipe/commit/0f6eeeb5af6b0ace3d8899236056232437891902)
- [upgrade material UI](https://github.com/ezcater/recipe/commit/a495b5adb094116ba2a4d83bfdf355a9938e2bc0)
- [fix console warning for color palette](https://github.com/ezcater/recipe/commit/2bde19de2c8c18a84311a421ad73bd29077d4145)
- [remove unused color token components](https://github.com/ezcater/recipe/commit/0c86bb33cc3ebfbd55de5871c83f2e82054028c9)

## Why are we doing this?
Initial work to set up design tokens in Recipe themes. This includes documentation. This does not update any of our components to use these new design tokens. That will be done in a separate PR.

## Screenshot(s) / Gif(s):
Screenshot 2023-09-22 at 2 48 12 PM

## Checklist

- [x] Create a changeset (`yarn changeset`) with a summary of the changes

### CODE REVIEW

File: packages/recipe/package.json [hunk 1]
```diff
@@ -27,8 +27,8 @@
     "@fortawesome/free-regular-svg-icons": "^6.4.0",
     "@fortawesome/free-solid-svg-icons": "^6.1.1",
     "@fortawesome/react-fontawesome": "^0.1.18",
-    "@mui/lab": "^5.0.0-alpha.106",
-    "@mui/material": "^5.10.12",
+    "@mui/lab": "^5.0.0-alpha.145",
+    "@mui/material": "^5.14.10",
```
Comment (danidewitt) [inline packages/recipe/package.json:31]: i assume this minor update doesn't cause us any breaking changes

File: packages/recipe/docs/components/DesignTokens/DesignTokensDisplay.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    {withCopy && (
+      <Box position="absolute" top="12px" right="12px">
+        <CopyButton darkMode textToCopy={children} />
+      </Box>
+    )}
+  </Box>
+);
+
+const DesignTokenPreview = ({token, tokenType}) => {
+  // TODO: add remaining tokens
```
Comment (danidewitt) [inline packages/recipe/docs/components/DesignTokens/DesignTokensDisplay.tsx:27]: is this for a different PR? fine to keep just wondering

File: packages/recipe/docs/components/DesignTokens/DesignTokensDisplay.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+      <Box>{previewMap[tokenType].displayText}</Box>
+    </Stack>
+  );
+};
+
+const DesignTokensDisplay = ({tokens, tokenType}: DesignTokensDisplayProps) => (
+  <Table aria-label="design tokens display">
+    <TableHead>
+      <TableRow>
+        <TableCell sx={{color: '#2e343899', fontWeight: 'bold'}}>Name</TableCell>
```
Comment (danidewitt) [inline packages/recipe/docs/components/DesignTokens/DesignTokensDisplay.tsx:56]: should the repeated colors be a variable?

Comment (danidewitt) [reply packages/recipe/docs/components/DesignTokens/DesignTokensDisplay.tsx:56]: actually the table cell as a whole is repeated.

File: packages/recipe/docs/components/DesignTokens/DesignTokens.tsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
+import React, {FC, useState} from 'react';
+import {Box, Tab, Tabs, useTheme} from '@mui/material';
+import {DesignTokensPanelProps} from './DesignTokens.types';
+import {EzThemeTokens} from '../../../src/themes/themes.types';
+import DesignTokensDisplay from './DesignTokensDisplay';
+
+// TODO: add remaining tokens
```
Comment (danidewitt) [inline packages/recipe/docs/components/DesignTokens/DesignTokens.tsx:7]: same as the other comment about todos

**Review by danidewitt — COMMENTED** [review]
some minor comments/questions

**Review by danidewitt — APPROVED** [review]

File: packages/recipe/docs/components/DesignTokens/DesignTokensDisplay.tsx (hunk 3 above)
Comment (danidewitt) [reply packages/recipe/docs/components/DesignTokens/DesignTokensDisplay.tsx:56]: i'll update my comments: this is really up to you, could be consolidated but not mandatory for merging

File: packages/recipe/package.json (hunk 1 above)
Comment (noranda) [reply packages/recipe/package.json:31]: Correct! Confirmed with Chromatic.

File: packages/recipe/docs/components/DesignTokens/DesignTokensDisplay.tsx (hunk 2 above)
Comment (noranda) [reply packages/recipe/docs/components/DesignTokens/DesignTokensDisplay.tsx:27]: Yep, I'm breaking these out into separate PRs.

File: packages/recipe/docs/components/DesignTokens/DesignTokensDisplay.tsx (hunk 3 above)
Comment (noranda) [reply packages/recipe/docs/components/DesignTokens/DesignTokensDisplay.tsx:56]: Usually yes, we want to use theme colors for our components, but this is an internal docs component that we don't expose. The styles here were added to match Storybook's color palette table headers that we use in the next section for consistency.

File: packages/recipe/docs/components/DesignTokens/DesignTokens.tsx (hunk 4 above)
Comment (noranda) [reply packages/recipe/docs/components/DesignTokens/DesignTokens.tsx:7]: Will be coming in future PRs.

File: packages/recipe/docs/components/DesignTokens/DesignTokensDisplay.tsx (hunk 3 above)
Comment (danidewitt) [reply packages/recipe/docs/components/DesignTokens/DesignTokensDisplay.tsx:56]: i meant like, at minimum, maybe we can define them as consts at the top of the file. or maybe we define them as internal docs variables.

Comment (noranda) [reply packages/recipe/docs/components/DesignTokens/DesignTokensDisplay.tsx:56]: I don't think that's necessary here as we'll likely replace this at some point with our new `EzTable`. This did not work well with our current `EzTable` which is why I opted to use MUI. Again, this is all internal and can be updated later without worrying about any breaking changes.

Comment (noranda) [reply packages/recipe/docs/components/DesignTokens/DesignTokensDisplay.tsx]: Made the change anyways. 👍

**Review by pgrippi — APPROVED** [review]

**Review by danidewitt — APPROVED** [review]

**Pull request merged**